assets/sass/components/_card.scss

```diff
@@ -0,0 +1,100 @@
+/* ==========================================================================
```

Comment:
Is this 'card' component a pattern they use in other services?

Reply:
Yes - I stole it from book a video link, but its in several others.